### PR TITLE
Removing grabcut and classic segmentations

### PR DESCRIPTION
**Pull request by alexdesiqueira**
In this PR, we remove ~classical~classic (Otsu threshold) and grabcut segmentation options, leaving only U-nets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,9 +70,9 @@ Resulting files:
 
 Running the command
 ```
-$ python pipeline.py -p -u -i ../mothra-data -o ../test_output -csv ../test_output/results.csv
+$ python pipeline.py -p -i ../mothra-data -o ../test_output -csv ../test_output/results.csv
 ```
-in `/mothra` will run the pipeline using the U-net deep neural network on the example data in the folder `/mothra-data`. The file locations should look like this:
+in `/mothra` will run the pipeline on the example data in the folder `/mothra-data`. The file locations should look like this:
```
Comment (stefanv) [inline README.md:75]: Should this be `./motha` to indicate relative to the root?

**Review by stefanv — APPROVED** [review]
Looks good!

Question: is there graceful fallback when the NN model file is not found?

Go ahead and merge when ready.

**Comment by alexdesiqueira** [comment]
Hey @stefanv,
> Question: is there graceful fallback when the NN model file is not found?

It downloads the files automatically:

```bash
❯ ./pipeline.py -u
(...)
Image 1/1 : BMNHE_500607.JPG
models/segmentation.pkl not in the path. Downloading...
Downloading data from 'https://gitlab.com/mothra/mothra-data/-/raw/main/models/segmentation/segmentation.pkl' to file '/home/alex/projects/mothra/models/segmentation.pkl'.
(...)
Identifying gender...
models/id_gender.pkl not in the path. Downloading...
Downloading data from 'https://gitlab.com/mothra/mothra-data/-/raw/main/models/id_gender/id_gender.pkl' to file '/home/alex/projects/mothra/models/id_gender.pkl'.
```
You're right, though; I hard-coded where the files should be. I think it'd be nice to have the option to pass the weights, and only then offer to download them.

**Comment by stefanv** [comment]
I think that's fine for now.

**Comment by alexdesiqueira** [comment]
Thank you @stefanv!

**Pull request merged**